### PR TITLE
Issue 948  radon scaling

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: odl/tomo/backends/astra_cuda.py [hunk 1]
```diff
@@ -281,7 +281,8 @@ def call_backward(self, proj_data, out=None):
         out[:] = self.out_array
 
         # Fix inconsistent scaling in ASTRA
```
Comment (kohr-h) [inline odl/tomo/backends/astra_cuda.py]: Can we remove these comments? It's largely been reduced to different conventions, no? No more need to be cocky.

File: odl/tomo/operators/ray_trafo.py [hunk 2]
```diff
@@ -164,16 +165,23 @@ def __init__(self, discr_domain, geometry, **kwargs):
...
+                             discr_domain.cell_volume)):
+                # Approximate cell volume
+                # TODO: angles and detector must be handled separately. While the
+                # detector should be uniformly discretized, the angles do not have
+                # to and often are not.
+                extent = float(geometry.partition.extent.prod())
+                size = float(geometry.partition.size)
+                weighting = extent / size
+            else:
+                raise ValueError('unknown weighting of domain')
```
Comment (kohr-h) [inline odl/tomo/operators/ray_trafo.py]: print something more useful

File: odl/test/largescale/tomo/ray_transform_slow_test.py [hunk 3]
```diff
@@ -172,7 +179,54 @@ def projector(request, dtype):
...
+    proj_adj = projector.adjoint(proj)
+    proj_adj_adj_adj = projector.adjoint.adjoint.adjoint(proj)
+
+    # Verify A^*(y) == ((A^*)^*)^*(x)
+    assert proj_adj == proj_adj_adj_adj
+
+
+@skip_if_no_largescale
 def test_reconstruction(projector):
+    return
```
Comment (kohr-h) [inline odl/test/largescale/tomo/ray_transform_slow_test.py]: left-over?

File: odl/tomo/operators/ray_trafo.py (hunk 2 above)
Comment (adler-j) [inline odl/tomo/operators/ray_trafo.py]: Frankly, no. This case currently cannot happen, so I'll change it to `NotImplementedError` and leave it at that.

File: odl/test/largescale/tomo/ray_transform_slow_test.py (hunk 3 above)
Comment (adler-j) [inline odl/test/largescale/tomo/ray_transform_slow_test.py]: yes

**Comment by adler-j** [comment]
Merge after CI.

**Pull request merged**